src/App/IntroPageEmployee/IntroPageEmployee.java

```diff
@@ -0,0 +1,214 @@
+package App.IntroPageEmployee;
+
+import App.Connect;
+import App.ProjectSummary.ProjectSummaryController;
+import App.SearchProject.SearchProject;
+import com.jfoenix.controls.JFXButton;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXMLLoader;
+import javafx.fxml.Initializable;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.scene.control.Label;
+import javafx.scene.control.TableColumn;
+import javafx.scene.control.TableView;
+import javafx.scene.control.cell.PropertyValueFactory;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.stage.Stage;
+
+import java.net.URL;
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.Statement;
+import java.util.ResourceBundle;
+import App.IntroPageAdmin.Project;
+
+public class IntroPageEmployee implements Initializable {
+
+    public JFXButton allProjectsBtn;
+    public JFXButton searchProjectBtn;
+    public JFXButton logoutBtn;
+
+    private int empId;
+
+    private String UserRole;
+
+    public String getUserRole() {
+        return UserRole;
+    }
+
+    public void setUserRole(String userRole) {
+        UserRole = userRole;
+    }
+
+    public int getEmployeeId() {
+        return empId;
+    }
+
+    public void setEmployeeId(int empId) {
+        this.empId = empId;
+    }
+
+    public Label nameUser;
+    public Label projectCountLabel;
+    public TableView<Project> projectTableView;
+
+    public TableColumn<Project,String> projectName;
+    public TableColumn<Project,String> projectStartDate;
+    public TableColumn<Project,String> projectEndDate;
+
+
+    private int projectCount;
+
+    private Stage stage;
+
+    public void getProjectTableData(){
+        //Clear the all column data from table
+        projectTableView.getItems().clear();
+        projectCount = 0;
+
+        try {
+            Connect connect =new Connect();
+            Connection connection=connect.getConnection();
+
+            Statement statement = connection.createStatement();
+            String sql = "SELECT distinct PROJECT_INFO.id, project_name, start_date, end_date, estimated_time FROM PROJECT_INFO, PROJECT_TASK WHERE PROJECT_INFO.id = PROJECT_TASK.id AND assigned = (SELECT name FROM EMPLOYEE WHERE EMPLOYEE.id="+getEmployeeId()+")";
```

Comment:
There is an error with project filter query for employee. I have added a commit 5fe097a fixing that issue. Update the command according to commit 5fe097a.

Reply:
or you can just select `allow edit from maintainers`. I will do the rest.